cmd/clusterctl/examples/vsphere/machines.yaml.template

```diff
@@ -14,29 +14,21 @@ items:
           apiVersion: "vsphereproviderconfig/v1alpha1"
           kind: "VsphereMachineProviderConfig"
           machineSpec:
-            datacenter: ""
-            datastore: ""
-            resourcePool: ""
+            datacenter: "${VSPHERE_DATACENTER}"
+            datastore: "${VSPHERE_DATASTORE}"
+            resourcePool: "${VSPHERE_RESOURCE_POOL}"
+            vmFolder: "${VSPHERE_FOLDER}"
             networks:
-            - networkName: ""
+            - networkName: "${VSPHERE_NETWORK}"
               ipConfig:
                 networkType: static
-                ip: ""
-                netmask: ""
-                gateway: ""
-                dns:
-                - xxxx
-                - yyyy
             numCPUs: 2
             memoryMB: 2048
-            template: ""
+            template: "${VSPHERE_TEMPLATE}"
             disks:
-            - diskLabel: ""
+            - diskLabel: "${VSPHERE_DISK}"
               diskSizeGB: 20
             preloaded: false
-            ntpServers:
-            - 0.pool.ntp.org
-            - 1.pool.ntp.org
       versions:
         kubelet: 1.11.3
```

Comment:
Hi @andrewsykim,

Note to selves -- we need to update the default kubelet and controlPlane versions to 1.13.6.

Reply:
template the k8s version too?

Reply:
That makes sense. With the default being whatever version we support in the images.